src/partials/contacts.html

```diff
@@ -1,69 +1,77 @@
-<div class="contacts">
+<div class="conntacts">
   <h2 class="is-hidden"></h2>
```

Comment:
ten niewidzialny tytuł jest pusty a powinien zazywać sie 

<h2 class="is-hidden">Contacts</h2>